### DISCUSSION TITLE
Websocket event notification

### DISCUSSION THREAD

**Discussion by swex** [Motion-Project/motion#1493] [General]
0. Reviewed guide and contributing documents? (Yes/No):yes
1. version [x.y.z, hash, other]: master
2. installed as a package or compiled from sources [deb, rpm, git, other]: NA
3. standalone or part of third party [motion, MotionEyeOS, other]: NA
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: NA
5. hardware [x86, ARM, other]: NA
6. operating system [16.04, Stretch, etc, FreeBSD, other]: NA

Current execute on event system is ok, but modern way would be to have websocket  endpoint echoing all events.

**Comment by brianolson** [comment]
make the on_event_start/on_... script called be a `curl` command to a web service?
